### ISSUE TITLE
Implement scraper worker for unshortening URLs

### ISSUE THREAD

**Issue by glciampaglia** [glciampaglia/osome-csdl-ranking-challenge#6]
# Tasks
- [ ] Scraper worker that receives batch unshortening tasks and stores results back in postgres. No caching used here. (Already updated [unshorten-fast](/glciampaglia/unshorten-fast)).
- [ ] Figure out if results can be sent back directly via celery or not 
- [ ] (If no results can be sent back via celery) Sandboxed worker that syncs Redis DB with Postgres unshortening results table
- [ ] Update AD batch scorer to make scraping request and use either Redis or Postgres for cached results

# Description

Should get tasks from celery app, put results in DB. The scorer should first check if short URLs are present in the DB, if not request unshortening via celery to scraper worker.

**Comment by glciampaglia** [comment]
After reading again the [infrastructure document](https://docs.google.com/document/d/1GaWmbojaEO0aUcHyb269lJAIzMtUbNcDo7pf5JHx7_M/edit) and looking at the example code of the scraper worker on the [HumanCompatibleAI/ranking-challenge](/HumanCompatibleAI/ranking-challenge) repository, I think the above is not feasible. 
* The scraper can access the postgres DB but only its own table and nothing else.
* The sandboxed worker on the other hand can read from the postgres DB of the scraper, but cannot write to it.
* And finally even though the sandboxed worker has access to Redis, the scraper does not.

In other words there is no way to communicate *which* URLs embedded in the post payload needs to be unshortened. It looks like this is an example of exfiltration that perhaps could be allowed under a more controlled access, like the Perspective API.

Posted another request for clarifications on the Discord of the organizers.
